src/more_ops.rs

```diff
@@ -73,53 +73,6 @@ fn limbs_for_int(v: &Number) -> usize {
     ((v.bits() + 7) / 8) as usize
 }
 
-fn u32_from_u8(mut buf: &[u8]) -> Option<u32> {
```

Comment:
this function (along with its tests) was moved into `op_utils.rs` as it seemed like a more appropriate place now that it's used in more places. It also changed semantics.

It was changed to return a *signed* int 32 and *reject* values that are too wide, even if there are leading zeros.

The rationale for not stripping leading zeros is that it would have a cost doing so, which means we would have to charge for that cost. By instead just rejecting such arguments, we don't need to charge a cost, since it's fixed at 4 bytes (or fewer)